.appveyor.yml

```diff
@@ -0,0 +1,79 @@
+# AppVeyor configuration for CI Builds of ICU4C.
+
+image: Visual Studio 2017
+platform: x64
+
+# Don't clone the entire repo.
+clone_depth: 1
+
+# Cache things between builds to speed things up and save bandwidth.
+cache:
+  - c:\icu-ci-cache
+
+# Notes:
+#  - The VS2017 build is disabled (commented out) below, as we build both VS2015 and VS2017
+#    on Azure Pipelines instead, since that is much faster.
+#
+#  - For the Cygwin build, the CYG_MIRROR URL below is from the "Cygwin Time Machine" archive,
+#    and that URL maps to the 64-bit version of Cygwin 2.11.2-1 (with timestamp 1550062412).
+#    This was the last release of Cygwin 2.11 before version 3.0.0 was released.
+#    See: http://www.crouchingtigerhiddenfruitbat.org/Cygwin/timemachine.html
+#
+#    Note: The archived Cygwin repo does not keep or retain any file signature files, so we
+#    must use the "-X" or "--no-verify" option below in the setup command.
+
+environment:
+  global:
+    ICU_CI_CACHE: c:\icu-ci-cache
+    CYG_URL: https://cygwin.com/setup-x86_64.exe
+    CYG_MIRROR: http://ctm.crouchingtigerhiddenfruitbat.org/pub/cygwin/circa/64bit/2019/02/13/045332
```

Comment:
Note: This will fail as the Cygwin time machine is down.

Reply:
and maven fails also … don't think it should be a blocker